TF_Tooling_Web/jest.config.js

```diff
@@ -1,4 +1,5 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
     preset: 'ts-jest',
-    testEnvironment: 'node',
+    testEnvironment: 'jsdom',
```

Comment:
💬 How come we switched to jsdom (just out of interest)

Reply:
Require to use things like `EventTarget` - needs to mimic a browser environment rather than Node